package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -128,7 +147,24 @@ export const SpotlightTourProvider = forwardRef<SpotlightTour, SpotlightTourProv
 
   const start = useCallback((): void => {
     renderStep(0);
-  }, [renderStep]);
+    onStart?.();
+  }, [renderStep, onStart]);
+
+  const startOnce = useCallback(() => {
+    if (!tourId) {
+      setTourId(hash(steps));
```

Comment:
What happens if the user has multiple tour providers in their app. I think we will need them to provide an ID of the tour to use this feature, and the value we check against is the `hash` of the steps. What do you think? 🤔